scripts/verification.py

```diff
@@ -27,20 +27,38 @@
 #
 # For more information, please refer to <http://unlicense.org/>
 #
-# Internal error codes:
+# Error codes:
 # -1 - no final newline error (one newline to few)
 #  0 - Everything fine
 #  1 - undefinded final newline error (newline not detectable)
-#  2 - one final newline to much error (2 newlines)
+#  2 - one final newline too much error (2 newlines)
+# 12 - tab error and no final newline error
 # 13 - tab error (unlucky 13)
+# 14 - tab error and undefinded final newline error
+# 15 - tab error and one final newline too much error
+# 20 - alphabetical order error and no final newline error
 # 21 - alphabetical order error (wrong order hence 21)
+# 22 - alphabetical order error and undefinded final
+#      newline error
+# 23 - alphabetical order error and one final newline too
+#      much error
+# 33 - alphabetical order error and tab error and no final
+#      newline error
+# 34 - alphabetical order error and tab error
+# 35 - alphabetical order error and tab error and undefinded
+#      final newline error
+# 36 - alphabetical order error and tab error and one final
+#      newline too much error
 # 42 - file not openable/readable error (this could mean
 #      anything hence 42)
+# 84 - file not openable/readable error on second import
+#      (same as 42 but twice as shady)
```

Comment:
Instead of manually crafting these numbers I would opt for powers of 2 for the "atomic" errors which can then be combined using bitwise operations.

Reply:
As those error codes have no further meaning for the CI since it only cares for non-zero exit codes I don't think we need to improve them. I will gladly come back to this topic if the verification script will be used in another step so that you need easier handling for machines.

Reply:
Additionally I removed 84 as it doesn't apply anymore since 7e1ed3aaa92a991e64e133448109cd17d73e2d60